python/dgl/nn/pytorch/conv/graphconv.py

```diff
@@ -205,6 +349,9 @@ def forward(self, graph, feat, weight=None):
             :math:`(N_{out}, D_{in_{dst}})`.
         weight : torch.Tensor, optional
             Optional external weight tensor.
+        edge_weight : torch.Tensor, optional
+            Optional tensor on the edge. If given, the convolution will weight
+            with regard to the edge feature.
```

Comment:
If you only want to support scalar edge weights, mention that.

Reply:
We support all broadcastable dimensions.

Reply:
In principle this can be broadcastable multidimensional edge weights like below.

```python
import dgl
import dgl.function as fn
import torch

g = dgl.graph(([0, 1], [1, 2]))
g.ndata['h'] = torch.randn(3, 2)
g.edata['w'] = torch.randn(2, 3, 1)
g.update_all(fn.u_mul_e('h', 'w', 'm'), fn.sum('m', 'h2'))
```

Reply:
I think this is exactly what I have in code